### PR TITLE
fix unclose menu issue

### PR DESCRIPTION
**Pull request by crackwebai1024**

### CODE REVIEW

**Review by nrutledge — CHANGES_REQUESTED** [review]
Build is failing due to the `seabug-sdk` submodule. It was removed on the main branch and shouldn't be added back. Please remove the submodule.

File: src/components/UI/molecules/AuctionCard/index.module.scss [hunk 1]
```diff
@@ -11,6 +11,15 @@
   break-inside: avoid;
   max-width: 100%;
 
+  .memb {
```
Comment (nrutledge) [inline src/components/UI/molecules/AuctionCard/index.module.scss:14]: What does `memb` mean? Is this supposed to be `menu`?

**Review by nrutledge — CHANGES_REQUESTED** [review]

File: src/components/UI/molecules/AuctionCard/index.module.scss (hunk 1 above)
Comment (crackwebai1024) [reply src/components/UI/molecules/AuctionCard/index.module.scss:14]: It is the meaning of membrane.

**Review by nrutledge — APPROVED** [review]
Looks good.

**Pull request merged**